### PR TITLE
GameDB: add patches to the 'K-1 World' series games add EE clamping full to 'D1 Professional Drift Grand Prix Series'

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

adds patches to the 'K-1 World' series games and adds ee clamping  full to  'D1 Professional Drift Grand Prix Series'
### Rationale behind Changes

Fixes said games 
### Suggested Testing Steps

look at CI/ try said games

### CODE REVIEW

File: bin/GameIndex.yaml [hunk 1]
```diff
@@ -14788,6 +14788,14 @@ SLES-53805:
 SLES-53809:
   name: "K-1 Premium Dynamite!!"
   region: "PAL-E"
+  gameFixes:
+    - XGKickHack # Fixes colors in fighters.
+  patches:
+    B6F9C8D3:
+      content: |-
+        comment= Fixes black screen in hardware mode.
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:14796]: remove space after comment=

File: bin/GameIndex.yaml [hunk 2]
```diff
@@ -21135,6 +21143,14 @@ SLPM-62580:
 SLPM-62581:
   name: "K-1 Premium 2004 - Fighting Dynamite!!"
   region: "NTSC-J"
+  gameFixes:
+    - XGKickHack # Fixes  colors in fighters.
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:21147]: "Fixes black and white fighters." be careful with your space bar.

File: bin/GameIndex.yaml [hunk 3]
```diff
@@ -24962,6 +24978,14 @@ SLPM-66077:
   name: "K-1 World Max 2005"
   region: "NTSC-J"
   compat: 5
+  gameFixes:
+    - XGKickHack # Fixes colors in fighters.
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:24982]: "Fixes black and white fighters."

File: bin/GameIndex.yaml [hunk 4]
```diff
@@ -31113,6 +31143,14 @@ SLPS-25577:
 SLPS-25578:
   name: "K-1 World Grand Prix 2005"
   region: "NTSC-J"
+  gameFixes:
+    - XGKickHack # Fixes colors in characters.
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:31147]: "Fixes black and white fighters."

**Comment by gilderoylockhart01** [comment]
Those games might also require instantVU1 to be disabled to fix flickering, but this needs testing before confirming.

File: bin/GameIndex.yaml [hunk 5]
```diff
@@ -24312,6 +24340,8 @@ SLPM-65862:
 SLPM-65863:
   name: "D1 Grand Prix Series - Professional Drift"
   region: "NTSC-J"
+  clampModes:
+   eeClampMode: 3 # Fixes black void
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:24344]: indenting

Comment (refractionpcsx2) [reply bin/GameIndex.yaml:24344]: also period on the end

**Pull request merged**